homeworks/HW3-final/model_performance.py

```diff
@@ -0,0 +1,41 @@
+import Regression as reg
+import numpy as np
+from sklearn import datasets
+from sklearn.model_selection import train_test_split
+from sklearn.linear_model import Ridge
+import matplotlib.pyplot as plt
+
+dataset = datasets.load_boston()
+X_train, X_test, y_train, y_test = train_test_split(dataset['data'], 
+                                                    dataset['target'], 
+                                                    test_size=0.2, 
+                                                    random_state=42)
+
+
+alpha_list = np.arange(0.05,1,0.09)
+score_list = []
+for a in alpha_list:
+    #model = reg.RidgeRegression(alpha)
+    model = reg.RidgeRegression(0.1)
```

Comment:
You don't want to do this in the for loop - you're creating lots of unnecessary RidgeRegression objects.